### PR TITLE
simplify custom CA with kubeResourceOverride

### PR DESCRIPTION
**Pull request by totallyGreg**
# Description

- Removed the kustomize patching in favor of providing the helm values
- Removed the Upgrade section as I believe there is no longer a reason to patch an existing pod

# Context

Trying to simply the process of bringing in a custom CA 

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/content/installation/advanced_configuration/extauth_custom_ca.md [hunk 1]
```diff
@@ -48,166 +48,54 @@ helm repo update
...
-patchesStrategicMerge:
-- custom-ca-patch.yaml
-EOF
-```
-
-Finally, we'll install Gloo Edge Enterprise with Helm using kustomize to add our patch in. Be sure to update the value for the license key.
-
-```bash
-helm install gloo glooe/gloo-ee --namespace gloo-system \
+helm update --install gloo glooe/gloo-ee --namespace gloo-system \
```
Comment (artberger) [inline docs/content/installation/advanced_configuration/extauth_custom_ca.md:90]:
```suggestion
helm upgrade --install gloo glooe/gloo-ee --namespace gloo-system \
```

**Review by artberger — COMMENTED** [review]
A couple doc suggestions, thanks!

**Comment by sam-heilbron** [comment]
LGTM! Leaving open for @artberger to give this another pass.

**Review by artberger — APPROVED** [review]

**Pull request merged**